README.md

```diff
@@ -169,7 +170,6 @@ Usage: mix coveralls.post <Options>
     -s (--sha)          Commit SHA (required when not using Travis)
     --build             Service number ('BUILDS' column at coveralls.io page)
     --parallel          coveralls.io 'parallel' option (See coveralls.io API Reference)
-```
```

Comment:
still need this to end the code block :)

Reply:
Closed :) Looked at the diff and it seems clean